src/Authentication/Authentication/Helpers/HttpHelpers.cs

```diff
@@ -54,7 +54,6 @@ private static HttpClient GetGraphHttpClient(AzureIdentityAccessTokenProvider au
                 new NationalCloudHandler(),
                 new ODataQueryOptionsHandler(),
                 new HttpVersionHandler(),
-                new CompressionHandler(),
```

Comment:
If you drop the handler, you also want to update the Decompression flags on line 72 similar to the library at https://github.com/microsoft/kiota-dotnet/blob/e891593e66ab61c5f9f820cdd31c433a2164aa67/src/http/httpClient/KiotaClientFactory.cs#L193